src/server/common.h

```diff
@@ -109,6 +105,33 @@ struct OpArgs {
   }
 };
 
+// A strong type for a lock tag. Helps to disambiguide between keys and the parts of the
+// keys that are used for locking.
+class LockTag {
```

Comment:
this must be in common because conn_context does not include "table.h"

Reply:
Yes and no 🙂 We hold ScriptInfo behind a unique_ptr, so it could've been forward declared... but it's a nit. Borys wanted to split common.h anyways 😆